docs/walkthrough/redshift-estimator.rst

```diff
@@ -0,0 +1,470 @@
+            self.max_val = 0.4
+            self.num_bins = 180
+            return
+
+
+        def execute(self, images, labels):
+            print(type(labels))
+            print("Initializing Model")
+
+            # Initialize the model
+            self.arch.compile(loss='sparse_categorical_crossentropy',
+                            optimizer='adam',
+                            metrics=['sparse_categorical_accuracy'])
+            print("Model Initialized Successfully")
+            
+            print("Beginning Training")
+            print("Training images shape:", images.shape)
+            print("Training labels shape:", labels.shape)
+
+            # Train the model on the images and the labels. Labels are converted to categorical
+            # data because the architecture expects an index to an output vector of length 180
+            self.arch.fit(images,
+                          self.to_categorical(labels),
+                          epochs=self.epochs,
+                          verbose=2)
+
+            print("Training Complete")
+
+            # Saves the model in a new variable. This is necessary so that the
+            # ouput of the operation is named 'model'
+            model = self.arch
+            return model
+
+        # Converts floating point labels to categorical vectors. The result for a given input
+        # label is a 1D vector of length 1 whose value is the index representing the range in
+        # which the label falls. For example, if the max_val is 0.4 and the num_bins is 4, the
+        # possible indices are 0-3, representing the ranges [0,0.1), [0.1,0.2), [0.2,0.3), and
+        # [0.3,0.4] respectively. So, a label of 0.12 results in an output of [1]
+        def to_categorical(self, labels):
+            return np.array(labels) // (self.max_val / self.num_bins)
+
+After the operation is fully defined, it needs to be added to the workspace and connected to the **Input** operations as shown below. Specifically, the training images and training outputs should be connected to the *images* and *labels* inputs of **TrainRedshift** respectively.
+
+Note that the architecture selected from within the pipeline editor until after the `Neural Network Architecture`_ section of this guide is completed.
+
+.. figure:: images/redshift-t.png
+    :align: center
+    :scale: 50%
+
+Neural Network Architecture
+===========================
+This section will describe how to create a Convolutional Neural Network for estimating redshift from images. In particular, this section gives instructions on creating an `Inception-v1 network <https://towardsdatascience.com/a-simple-guide-to-the-versions-of-the-inception-network-7fc52b863202#8fff>`_. The basic structure of this network is an input block, a series of five inception blocks, followed by a densely connected classifier block. These blocks are each described in order below.
+
+For reference during design, the full architecture is shown below.
```

Comment:
The image is too small to be able to see. Can you right click the architecture editor's background, expand all nodes, then download as PNG? Then you can probably link to it directly in case anyone wants to view it.